src/actions/users.js

```diff
@@ -224,6 +224,8 @@ export function getProfiles(offset, limit = General.PROFILE_CHUNK_SIZE) {
             return null;
         }
 
+        const profiles = profileListToMap(profilesList);
```

Comment:
It does an assign https://github.com/mattermost/mattermost-redux/blob/master/src/reducers/entities/users.js#L103